2021/29xxx/CVE-2021-29096.json

```diff
@@ -1,18 +1,146 @@
 {
```

Comment:
'Esri' does not match '^([a-zA-Z0-9_\\-\\.]+)@([a-zA-Z0-9_\\-\\.]+)\\.([a-zA-Z]{2,5})$'

Reply:
fixed